python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -0,0 +1,567 @@
+    * DryTest: Disable CSCs and unit tests
+    """
+
+    Setup = 1 << 3
+    Configure = 1 << 4
+    TakeFlat = 1 << 5
+
+    def __iter__(self) -> typing.Iterator[int]:
+        return iter(
+            [
+                self.All,
+                self.StateTransition,
+                self.MonitorState,
+                self.MonitorHeartBeat,
+                self.Setup,
+                self.Configure,
+                self.TakeFlat,
+                self.DryTest,
+            ]
+        )
+
+
+class MTCalsys(BaseCalsys):
+    """LSST Simonyi Telescope Calibration System.
+
+    MTCalsys encapsulates core functionality from the following CSCs:
+    TunableLaser, LEDProjector, LinearStage, Fiberspectrograph, Electrometer.
+
+    Parameters
+    ----------
+    domain : `lsst.ts.salobj.Domain`
+        Domain for remotes. If `None` create a domain.
+    log : `logging.Logger`
+        Optional logging class to be used for logging operations. If `None`,
+        creates a new logger. Useful to use in salobj.BaseScript and allow
+        logging in the class use the script logging.
+    intended_usage: `int`
+        Optional integer that maps to a list of intended operations. This is
+        used to limit the resources allocated by the class by gathering some
+        knowledge about the usage intention. By default allocates all
+        resources.
+    mtcamera: ComCam or LSSTCam
+    """
+
+    def __init__(
+        self,
+        domain: typing.Optional[salobj.Domain] = None,
+        log: typing.Optional[logging.Logger] = None,
+        intended_usage: typing.Optional[int] = None,
+        mtcamera: typing.Optional[ComCam] = None,
+    ) -> None:
+
+        self.electrometer_projector_index = 201
+        self.fiberspectrograph_blue_index = 1
+        self.fiberspectrograph_red_index = 2
+        self.linearstage_led_select_index = 1
+        self.linearstage_led_focus_index = 2
+        self.linearstage_laser_focus_index = 3
+        self.linearstage_select_index = 4
+
+        super().__init__(
+            components=[
+                "TunableLaser",
+                "LEDProjector",
+                f"FiberSpectrograph:{self.fiberspectrograph_blue_index}",
+                f"FiberSpectrograph:{self.fiberspectrograph_red_index}",
+                f"Electrometer:{self.electrometer_projector_index}",
+                f"LinearStage:{self.linearstage_led_select_index}",
+                f"LinearStage:{self.linearstage_led_focus_index}",
+                f"LinearStage:{self.linearstage_laser_focus_index}",
+                f"LinearStage:{self.linearstage_select_index}",
+            ],
+            domain=domain,
+            log=log,
+            intended_usage=intended_usage,
+        )
+
+        self.mtcamera = mtcamera
+        self.ls_select_led_location = 0.0  # um
+        self.ls_select_laser_location = 20.0  # um
+
+        self.laser_enclosure_temp = 20.0  # C
+
+        # ALl this needs to go in a config yaml file somewhere.
```

Comment:
any reason for not putting this in a configuration file already? You could add them to the same place we add the calibration configuration.

If you rather do this later, then please, create a ticket and add a TODO here.

Reply:
Mostly just wanted to check with you first about the location and format you'd like to see for something like this.